gui/wxpython/location_wizard/wizard.py

```diff
@@ -1481,13 +1479,12 @@ def OnEnterPage(self, event):
         event.Skip()
 
     def OnPageChanging(self, event):
-        if event.GetDirection():
+        isblank = bool(self.wktstring and self.wktstring.strip())
+        if event.GetDirection() and isblank is False:
```

Comment:
simply just `if event.GetDirection() and not self.wktstring.strip():` should suffice.